Documentation/cookbook_overview.rst

```diff
@@ -49,7 +51,7 @@ Input file resources
 ~~~~~~~~~~~~~~~~~~~~
 
 The files needed to work through the examples are provided. Users may
-download freely from this repository (https://github.com/daisy20170101/SeisSol_Cookbook).
+download freely from Cookbook repository: https://gitlab.lrz.de/ru58zid/benchmarks/
```

Comment:
The link is not available to the public.

Reply:
All files necessary to setup the cookbook examples can be downloaded at:

Reply:
Or I should use the old one: https://github.com/daisy20170101/SeisSol_Cookbook as the repository for all the materials?

Reply:
No gitlab is fine, you just need to make it available to the public (Project settings -> Visibility -> Public)